app/helpers/menu_helper/common_menu.rb

```diff
@@ -9,11 +9,18 @@ def initialize(user, context)
     end
 
     def items
-      [
-        item(t('menu.bill_of_materials'), context.bill_of_materials_path),
-        item(t('menu.items'), context.items_path),
-        item(t('menu.unit_of_measures'), context.unit_of_measures_path)
-      ]
+      menu_options.each_with_object([]) do |value, results|
```

Comment:
Why not just map it, rather than using each_with_object?

i.e.

```ruby
menu_options.map do |menu_option|
  item(t("menu.#{value}"), context.method("#{value}_path").call)
end
```